src/reportWebVitals.js

```diff
@@ -1,15 +0,0 @@
-const reportWebVitals = (onPerfEntry) => {
```

Comment:
- [ ] Nice work removing this file, but you still need to remove these other files in the public directory.

  ![image](https://github.com/Rysth/REACT-Math-Magicians/assets/81468675/161cb9ca-bcb2-46bf-a909-120d7b4667f0)

Reply:
Done!